src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -94,19 +105,22 @@ class AdditionalTab extends React.PureComponent<Props, State> {
         });
 
         const tooltipText = isDisabled ? this.getDisabledReason() : title;
+        const isFtuxTooltipVisible = !isLoading && !!ftuxTooltipData;
 
         return (
-            <Tooltip position="middle-left" text={tooltipText}>
-                <PlainButton
-                    className={className}
-                    data-testid="additionaltab"
-                    type="button"
-                    isDisabled={isDisabled}
-                    onClick={() => callbackFn({ id, callbackData: rest })}
-                >
-                    {this.getTabIcon()}
-                </PlainButton>
-            </Tooltip>
+            <AdditionalTabFtuxTooltip ftuxTooltipData={ftuxTooltipData} isVisible={isFtuxTooltipVisible}>
```

Comment:
Can this render multiple tooltips? The targeted one and then the "default" (`Tooltip`) one?

Reply:
Yeah, it'll render the default on the left side and the targeted one on the right:
<img width="523" alt="Screen Shot 2020-07-22 at 10 54 24 AM" src="https://user-images.githubusercontent.com/21277746/88211030-ea8b6f80-cc09-11ea-90f0-8d7721a59a5d.png">

I thought about suppressing the default one when there's a targeted tooltip, but I think we'd want to keep both for situations where the targeted toolip doesn't say the app name. For example, if we had a targeted tooltip for video conferencing apps that says something like "Working from home? Try out one of our video apps" and shows for either Zoom, Webex, or Hangouts, you'd still want to see the app name when you hover over the icon so you know which app you're launching

Reply:
Yeah I see what you're saying. I wonder if Design/Product have opinions on this. I think it looks like a lot with two tooltips, but I understand that's the only place you can see the application name aside from the logo.

Reply:
Just updated the PR - we decided to suppress the app name tooltip when the FTUX tooltip is showing